geckomat/change_model/applyKcatConstraints.m

```diff
@@ -80,90 +41,60 @@
 end
 
 %Clear existing incorporation of enzyme usage
+% TODO: modify for geckoLight 
```

Comment:
Doesn't the code below also solve this for GeckoLight? It is only the stoichiometry of one metabolite, the prot_pool, that needs to be cleared, and I think the code does that?

Reply:
I wasn't 100% certain how GeckoLight model would look like, so had kept this TODO to check later if it is needed. But you're right, the current code should handle it properly.